src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForDescriptorCondition.cs

```diff
@@ -46,15 +46,22 @@ protected virtual IMenuBarItem CreateAddNewNotMatchTagCondition()
             .WithActionCommand(() => _presenter.NewNotMatchTagCondition());
       }
 
+      protected virtual IMenuBarItem CreateAddNewInContainerCondition()
+      {
+         return CreateMenuButton.WithCaption(AppConstants.Captions.NewInContainerCondition)
+            .WithActionCommand(() => _presenter.NewInContainerCondition());
+      }
+
       public override IEnumerable<IMenuBarItem> AllMenuItems()
       {
          var allItems = new List<IMenuBarItem>
          {
             CreateAddNewMatchCondition(),
-            CreateAddNewNotMatchTagCondition()
+            CreateAddNewNotMatchTagCondition(),
+            CreateAddNewInContainerCondition()
```

Comment:
I am adding the InCOntainer condition all the time at the moment. To discuss

Reply:
I would keep it like this (adding everywhere). If user defines it somewhere where it does not make sense - so what, it's user's problem